rest/taskana-rest-spring/src/test/java/pro/taskana/classification/rest/ClassificationControllerRestDocTest.java

```diff
@@ -15,6 +16,7 @@
 import pro.taskana.common.rest.RestEndpoints;
 import pro.taskana.rest.test.BaseRestDocTest;
 
+@Disabled
```

Comment:
Where the tests disabled on purpose? (All the *RestDocTest)

Reply:
All Tests are disabled on purpose. The reason is the change to OpenApi instead of Spring RestDocs. The Tests will be fully removed with this PR, which will be integrated after this one:

https://github.com/Taskana/taskana/pull/2641/files#diff-50d989b4eee8fb31c35136ac140687f5a70ae6aed39dd9d1a508c17a66dad14f